packages/docs/site/i18n/gu/docusaurus-theme-classic/footer.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Some of those texts are used on the Docusaurus UI. You can translate the `message` property to Gujarati